src/components/Cell.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type CellProps = {
+    classes: {
+        cell: string
+    },
+    letter: null
+}
+
+export const Cell: React.FC<CellProps> = (props: CellProps) => {
+
```

Comment:
remove blank space

Reply:
You can do this automatically by making vscode format the code on save

Reply:
... and by having your prettier well configured of course

Reply:
Empty spaces where, inside the functions or everywhere? I have the habit of leaving some space between the imports and the code, and in the code as well since it makes it a little easier for me to read.